internal/uvm/create.go

```diff
@@ -70,6 +70,12 @@ type Options struct {
 	// ExternalGuestConnection sets whether the guest RPC connection is performed
 	// internally by the OS platform or externally by this package.
 	ExternalGuestConnection bool
+
+	// NetworkConfigProxy holds the address of the network config proxy service.
+	// This != "" determines whether to start the ComputeAgent TTRPC service
+	// that receives the UVMs set of NICs from this proxy instead of enumerating
+	// the endpoints locally.
+	NetworkConfigProxy string
```

Comment:
We should put this in the Options struct in cmd/containerd-shim-runhcs-v1/options/runhcs.proto so that it can be configured in containerd.toml.

Reply:
I think that will make it easier to configure for an environment rather than an option on UVM creation.

Reply:
I agree, this was just for easier local testing. I didnt think we were going to follow through with the annotation approach.

Reply:
Kept the annotation and followed how we handle things for the GPU vhd and others where if the annotation is provided it will override the containerd.toml setting.